app/workers/notification_worker.rb

```diff
@@ -3,7 +3,15 @@
 class NotificationWorker
   include Sidekiq::Worker
 
-  sidekiq_options queue: 'push', retry: 5
+  sidekiq_options queue: 'push', retry: 10, dead: false
+
+  sidekiq_retry_in do |count|
+    if count < 3
+      5 * (count + 1)
+    else
+      (count ** 4) + 15 + (rand(30) * (count + 1))
```

Comment:
`count ** 4` -> `count**4` (for RuboCop)

ref https://github.com/bbatsov/rubocop/issues/2972